IDEAS/Buildings/Components/Interfaces/PartialSurface.mo

```diff
@@ -56,7 +59,8 @@ partial model PartialSurface "Partial model for building envelope component"
   IDEAS.Buildings.Components.BaseClasses.ConductiveHeatTransfer.MultiLayer
     layMul(final inc=inc, energyDynamics=energyDynamics,
     linIntCon=linIntCon_a or sim.linearise,
-    A=A)
+    A=A,
+    errorLowPerformance = errorLowPerformance)
```

Comment:
I don't think that we need this parameter here? You can directly set multiLayer.errorLowPerformance = glazing.errorLowPerformance